lib/pages/guide.js

```diff
@@ -98,12 +123,11 @@ async function generate(text, layout, info) {
     .use(slug)
     .run(rawTextMdast);
 
-  // Add classes to the special sections in the mdast
-  tagCalloutSections(textMdastWithIds);
-
   // Generate the body text
   const bodyHast = await unified()
     .use(smartypants)
+    .use(directive)
```

Comment:
If we're going to use directives (which we're currently not in this PR), this needs to go up before `.use(md2mast)`.

Reply:
Hmm, that does make the other directives work. /shrug

Reply:
updated to use an image directive